arex-instrumentation-foundation/src/main/java/io/arex/foundation/model/DatabaseMocker.java

```diff
@@ -31,17 +33,18 @@ public DatabaseMocker(String dbName, String sql, String parameters) {
         this(dbName, sql, parameters, null, null);
     }
 
-    public DatabaseMocker(String dbName, String sql, String parameters, Object response) {
-        this(dbName, sql, parameters, null, response);
+    public DatabaseMocker(String dbName, String sql, String parameters, Object response, String keyHolder) {
+        this(dbName, sql, parameters, null, response, keyHolder);
     }
     
-    public DatabaseMocker(String dbName, String sql, String parameters, String tables, Object response) {
+    public DatabaseMocker(String dbName, String sql, String parameters, String tables, Object response, String keyHolder) {
```

Comment:
Too many arguments in constructor  remove it and use  'getKeyHolder' and 'setKeyHolder' directly.